### ISSUE TITLE
GHIssue returned by searchIssues API has null owner field

### ISSUE THREAD

**Issue by blindpirate** [hub4j/github-api#497]
### Context

`searchIssues` API returns a list of `GHIssue`s but these `GHIssue`s has `null` owner field. If anyone tries to invoke further API on these `GHIssue` (say, comment API), NPE will be thrown.

### Reproduction

```
List<GHIssue> issues = gitHub.searchIssues().q("repo:gradle/gradle").list().asList();
issues.get(0).getComments();
```

throws NPE.

### Workaround

Use reflection to set `owner` for the issues.

**Comment by t0m4uk1991** [comment]
@bitwiseman Looks like the bug already fixed and covered with [unit test](https://github.com/hub4j/github-api/blob/7a0f2fefd4f127aa004a492261521b983543cfd7/src/test/java/org/kohsuke/github/AppTest.java#L890)

Also, I wrote a [bug specific](https://github.com/hub4j/github-api/pull/1201) test (issue search in repo).

**Comment by bitwiseman** [comment]
You're correct, closing this issue.

**Comment by bitwiseman** [comment]
Fixed by #709
